tritium-metrics/src/main/metrics/metrics.yml

```diff
@@ -76,10 +72,6 @@ namespaces:
         type: timer
         tags: [executor]
         docs: A timer of the time it took a task to start running after it was submitted.
-      scheduled.once:
-        type: meter
-        tags: [executor]
-        docs: A meter of the number of one-shot scheduled tasks. Applies only to scheduled executors.
       scheduled.repetitively:
         type: meter
         tags: [executor]
```

Comment:
I'm not sure what this metric tells us, while we're breaking metrics, should this be removed? We could use `submitted` instead, or not record a metric in this case. Thoughts?

Reply:
If we're getting rid of `scheduled.once` we should also get rid of this metric. They're effectively the same metric just for one-shot vs repeated tasks.

Reply:
I've never really felt the need to distinguish between the two types of submissions (in my experience, one-shot scheduling is pretty rare) so I think just relying on `submitted` is sufficient.

Reply:
Might as well squash it into the `submitted` meter the same way as the `scheduled.once` metric I suppose?

Reply:
:+1:

Reply:
This is marked by calls to `scheduleAtFixedRate` & `scheduleWithFixedDelay`.  I'm not sure how folks are making use of this, especially as a `meter` (could probably be a `counter` if we want to keep it). I guess one might want to track recurring task submission rates. `scheduleAtFixedRate` calls are wrapped with a `TaggedMetricsScheduledRunnable` that tracks `scheduledOverrun`, but we don't currently have anything here tracking tasks recurring tasks that have thrown and would not be rescheduled.

Reply:
ahh, actually looks like this was directly ported over for compatibility with Dropwizard metrics https://github.com/dropwizard/metrics/blob/cb3c59e4c71b4df761f5809d56e00dd0d82ef7cb/metrics-core/src/main/java/com/codahale/metrics/InstrumentedScheduledExecutorService.java#L92-L101